newickFormatReader.py

```diff
@@ -94,7 +108,12 @@ def parse_newick(newick_string, tree_type):
     dfs_list = [(node.name, int(D[node.name])) for node in tree.find_clades()]
     tree_dict = {}
     build_tree_dictionary(build_tree(dfs_list), "Top", tree_dict, tree_type)
-    return tree_dict
+    real_distances = tree.depths()
+    real_D = {}
+    for clade in real_distances:
+        name = clade.name
+        real_D[name] = dist
+    return tree_dict, real_D
```

Comment:
I also think we should use `snake_case` for variable names (see [Google's python style guide](http://google.github.io/styleguide/pyguide.html#316-naming)).